languages/elixir/exercises/concept/tuples/.docs/introduction.md

```diff
@@ -0,0 +1,67 @@
+In Elixir, a tuple is a data structure which organizes data in contiguous memory. It behaves as a static data structure, holding a fixed number of items, but without explicit names for each element. Tuples are often used in Elixir for memory read-intensive operations, since read-access of an element is a constant-time operation. They are not usually used when elements may need to be added/removed dynamically because rather than modifying the existing tuple, a new tuple is created which requires memory to be allocated upfront.
```

Comment:
I must admit, I had to look up what "contiguous memory" means. Is it just my personal lack of basic knowledge, or could there be a more beginner-friendly way of saying the same thing?

Anyway, I think that that information and constant-time read access are very important characteristics of tuples and I would still keep them in the introduction. I personally had a hard time understanding why tuples and lists exist at the same time, and only learning about memory aspect of those made it clear to me.

Reply:
Okay! I'm not an Elixir programmer. Maybe then find some way to make this as simple as possible?